cfme/tests/infrastructure/test_instance_analysis.py

```diff
@@ -92,7 +96,7 @@ def pytest_generate_tests(metafunc):
                     continue
 
             # Set VM name here
-            vm_name = 'test_ssa_{}-{}'.format(fauxfactory.gen_alphanumeric(), vm_analysis_key)
+            vm_name = 'ssa_{}-{}'.format(fauxfactory.gen_alphanumeric(), vm_analysis_key)
```

Comment:
No, found no references to vm names in sprout logs, so its not a celery task.

It uses the same root login on vsphere55 and rhev34, so all I know is the timestamps